_opensearch/pipeline-agg.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
 title: Pipeline aggregations
-parent: Aggregations
-nav_order: 4
+parent: Aggregate data for analysis
```

Comment:
Can we move these files to the Search and Query directory?

Reply:
ok sure